### ISSUE TITLE
Export Keycloak for Import

### ISSUE THREAD

**Issue by F5Brewery** [AOT-Technologies/forms-flow-ai#48]
This project appears to be dependent on Keycloak and some specific settings.  I would highly suggest that you perform an Export from a good working Keycloak Realm instance and provide to the system installer as a template.  The instructions are missing some items and are a bit hard to follow.  A baseline template would be immensely helpful.  The instruction should stay and also improved as this does help one to understand some of it.

**Comment by richardsweeting** [comment]
Hi @F5Brewery we are holding a formsflow.ai workshop where we'll be addressing issues like this. Please feel free to sign up  - we'll make sure there is time for questions at the end! https://formsflow.ai/register/

**Comment by F5Brewery** [comment]
I would much rather see the actual issues addressed instead of a feature request.  My confidence in this sank dramatically once I got little reply.  I may attend, but sadly the folks interested in this pulled and requested I stop investing any time.

________________________________

**Comment by richardsweeting** [comment]
Yes I understand. I'm sorry to hear that you didn't get timely feedback. If there is anything else we can help with please let me know and I'll look into it for you.

**Comment by abhilash-aot** [comment]
We have taken the request for providing a import realm json for keycloak . This will be available from version 4. Thanks .
